pkg/controller/worker/machines_test.go

```diff
@@ -911,3 +918,11 @@ func addNameAndSecretToMachineClass(class map[string]interface{}, name string, c
 	}
 	class["secret"].(map[string]interface{})["labels"] = map[string]string{v1beta1constants.GardenerPurpose: genericworkeractuator.GardenPurposeMachineClass}
 }
+
+func makeCopyOfMap(mp map[string]string) map[string]string {
+	var mpDeepCopy = make(map[string]string)
+	for k, v := range mp {
+		mpDeepCopy[k] = v
+	}
+	return mpDeepCopy
+}
```

Comment:
this has been declared again in this file although it was declared in file `helper.go` as importing the package of `helper.go` makes lot of changes in code( the name of the imported package comes in front of many functions)

Reply:
Aren't both of these in the same package? It should be accessible without any imports then?

Reply:
no they aren't ...one is in `worker` package and the other is in `worker_test` package. and both the packages are in the `pkg/controller/worker`

Reply:
Sorry i forgot about that. Isn't something like this possible then - https://github.com/gardener/gardener-extension-provider-aws/blob/8eb133cbb838bc2598865e7a288e6d2900e7278a/pkg/controller/worker/machines_test.go#L403.